lib/stat_tracker.rb

```diff
@@ -40,4 +40,24 @@ def game_teams_generator(locations)
     end
   end
   
+  def highest_total_score
+    #@games to find all game scores and pick out the highest total score
+    
+    
+    # total = []
+    # @games.each do |game|
+    #   p game.game_id if game.away_goals.to_i + game.home_goals.to_i == 11
+    #   total << game.away_goals.to_i + game.home_goals.to_i
+    # end
+    # total.find do |score|
+    #   score == 11
+    # end
+
+    
```

Comment:
Nice. I love a good max_by. And we can probably just use take this and use min_by to get lowest_total_score, too!